core/src/main/scala/knobs/Parser.scala

```diff
@@ -98,7 +98,6 @@ object ConfigParser {
 
   lazy val ident: Parser[Name] = for {
     n <- satisfy(c => Character.isLetter(c)).map2(takeWhile(isCont))(_ +: _)
-    _ <- failWhen[Parser](n == "import", s"reserved word ($n) used as an identifier")
```

Comment:
I see it from `sansImport` if we do `n.mkString`, but since it never worked and wasn't tested, I'm comfortable with dropping it.